docs/github-labels.md

```diff
@@ -0,0 +1,116 @@
+| priority/backlog                   | Higher priority than priority/awaiting-more-evidence. | Any |
+| priority/critical-urgent           | Highest priority. Must be actively worked on as someone's top priority right now. | Any |
+| priority/important-longterm        | Important over the long term, but may not be staffed and/or may need multiple releases to complete. | Any |
+| priority/import-soon               | Must be staffed and worked on either currently, or very soon, ideally in time for the next release. | Any |
+| ready-to-work                      | Indicates that an issue or PR has been sufficiently triaged and prioritized and is now ready to work. | Any |
+| size/L                             | Denotes a PR that changes 100-499 lines, ignoring generated files. | Any |
+| size/M                             | Denotes a PR that changes 30-99 lines, ignoring generated files.| Any |
+| size/S                             | Denotes a PR that changes 10-29 lines, ignoring generated files.| Any |
+| size/XL                            | Denotes a PR that changes 500+ lines, ignoring generated files.| Any |
+| size/XS                            | Denotes a PR that changes 0-9 lines, ignoring generated files.| Any |
+| triage/duplicate                   | Indicates an issue is a duplicate of other open issue. | Humans |
+| triage/needs-information           | Indicates an issue needs more information in order to work on it. | Humans |
+| triage/not-reproducible            | Indicates an issue can not be reproduced as described. | Humans |
+| triage/unresolved                  | Indicates an issue that can not or will not be resolved. | Humans |
+
+## Labels that apply only to issues
+
+| Label | Description | Added By |
+|-------|-------------|----------|
+| good first issue                   | Denotes an issue ready for a new contributor, according to the "help wanted" [guidelines](issue-management.md#good-first-issues-and-help-wanted). | Anyone |
+| help wanted                        | Denotes an issue that needs help from a contributor. Must meet "help wanted" [guidelines](issue-management.md#good-first-issues-and-help-wanted). | Anyone |
+
+## Labels that apply only to PRs
+
+| Label | Description | Added By |
+|-------|-------------|----------|
+| approved                           | Indicates a PR has been approved by owners in accordance with [GOVERNANCE.md](../GOVERNANCE.md) guidelines. | Maintainers |
+| vmware-cla: no                     | Indicates the PR's author has not signed the (VMware CLA)[https://cla.vmware.com/faq] | VMware CLA Bot |
+| vmware-cla: yes                    | Indicates the PR's author has signed the (VMware CLA)[https://cla.vmware.com/faq] | VMware CLA Bot |
+| do-not-merge/hold                  | Indicates a PR should not be merged because someone has issued a /hold command | Merge Bot |
+| do-not-merge/work-in-progress      | Indicates that a PR should not merge because it is a work in progress. | Merge Bot |
```

Comment:
s/should not merge/should not be merged

Reply:
Corrected.

Reply:
Also, fixed the CLA link. User error, I reversed the '(' and '[' pairs.